app/main.py

```diff
@@ -187,6 +201,10 @@ def custom_openapi(openapi_prefix: str = ""):
     openapi_schema["x-tagGroups"] = [
         {"name": "Vector Tiles API", "tags": ["Dynamic Vector Tiles", "Vector Tiles"]},
         {"name": "Raster Tiles API", "tags": ["Raster Tiles"]},
+        {
+            "name": "Titiler COG Tiles",
+            "tags": ["Single COG Tiles", "Mosaic Tiles", "Custom Tiles"],
```

Comment:
For consistency, maybe you can go with all the tags having 'COG 'in them, or all the tags don't have 'COG' in them?